### DISCUSSION TITLE
DNNConnect.CKE for CKEditor 5

### DISCUSSION THREAD

**Discussion by SmartDev210** [dnnsoftware/Dnn.Platform#4875] [Q&A]
Do you have DNNConnect.CKE for CKEditor 5 or is it in progress of development.
I would really like to use CKEditor 5 in my DNN site.
Is there any way to use CKEditor 5 instead of CKEditor 4?

**Comment by bdukes** [comment]
It is not in progress. The license of CKEditor changed between version 4 and 5, so DNN Platform probably won't be able to include a CKEditor 5 provider out of the box.

**Comment by wbonekamp** [comment]
Same problem here... we really need an alternative HTML Editor...

So because of the license model, upgrading to CKEditor version 5 isn't an opion at all?
We really like CKEditor 5... does it make sense to give it another try?

If CKEditor 5 is not an option... is there an alternative we can have a look at?

**Comment by bdukes** [comment]
Someone in the community could build a CKEditor 5 editor provider, but DNN Platform can't take a direct dependency on it, in my understanding.
